library/std/src/sys/itron/abi.rs

```diff
@@ -0,0 +1,176 @@
+//! ABI for μITRON derivatives
+pub type int_t = crate::os::raw::c_int;
+pub type uint_t = crate::os::raw::c_uint;
+pub type bool_t = int_t;
+
+/// Kernel object ID
+pub type ID = int_t;
+
+/// The special value of `ID` representing the current task.
+pub const TSK_SELF: PRI = 0;
+
+/// Relative time
+pub type RELTIM = u32;
+
+/// Timeout (relative time or infinity)
+pub type TMO = u32;
+
+pub type HRTTIM = u32;
```

Comment:
missing comment?

Reply:
Addressed in 5eab31f7e97afed8c2215dd14b30d2f2962cf989.